plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/StreamTransferConsumer.java

```diff
@@ -238,7 +239,9 @@ public void fetchRow(DBCSession session, DBCResultSet resultSet) throws DBCExcep
                 targetRow[i] = value;
             }
             // Export row
-            processor.exportRow(session, resultSet, targetRow);
+            if (!(resultSet instanceof StatResultSet)) {
```

Comment:
Just omit rows export? I think we shoul;dnb't perform export for entire resultset.